pygame/time.py

```diff
@@ -112,7 +113,7 @@ def get_ticks():
 @ffi.callback("SDL_NewTimerCallback")
 def _timer_callback(interval, param):
     if sdl.SDL_WasInit(sdl.SDL_INIT_VIDEO):
-        event = ffi.new("SDL_Event")
+        event = ffi.new("SDL_Event*")
```

Comment:
It's copied internally by SDL in handling SDL_PushEvent, so we don't need to hang onto event once that completes.

(SDL_PushEvent calls SDL_PeekEvent (because of how SDL's event queue locking works, I believe) which calls SDL_AddEvent which copies the event)